src/MergeContext.js

```diff
@@ -4,6 +4,26 @@ const Log = require('./Logger.js');
 const GH = require('./GitHubUtil.js');
 const Util = require('./Util.js');
 
+// Contains properties used for approval test status creation
+class Approval {
+    constructor() {
+        this.reset();
+    }
+
+    reset() {
+        this.assign(null, null, null);
+    }
+
+    assign(aDesc, aState, aDelayMs) {
+        this.description = aDesc;
+        this.state = aState;
+        // If approved(slow burner): the number for milliseconds to wait for
```

Comment:
We should not consider PRs waiting on slow burner approved. Also, PRs on slow burner should be treated very similar to PRs on the fast track that are waiting for their timeout. but the proposed comment items lack that symmetry.

How about the following instead?

```
// If waiting for a timeout to merge the staged commit: > 0
// If ready to merge the staged commit now: == 0
// Otherwise (e.g., failed description, negative votes, or review requested): null
```